### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by jmaslanka55**
Merge

### CODE REVIEW

**Comment by jmaslanka55** [comment]
> This entire modal was designed by myself but was done over vscode live share so the commit occurred from Jason Maslanka's github.

This dude capping

File: lite-pm/pages/components/AppPage.js [hunk 1]
```diff
@@ -7,23 +7,22 @@ import {
   ModalBody,
   ModalFooter,
 } from "reactstrap";
-
+{
+}
```
Comment (CaioCamatta) [inline lite-pm/pages/components/AppPage.js]: What's this?

**Review by CaioCamatta — APPROVED** [review]

File: lite-pm/pages/components/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        </Container>
+
+        <div>
+          <this.renderCreateProjectModal/>
+        </div>
+      </div>
+    );
+  }
+
+}
```
Comment (Jaskanwar) [inline lite-pm/pages/components/HomePage.js:143]: Just want to put it out there that this modal was created by myself just happened to be over VS Code Live Share so Jason got the commit.

**Review by Jaskanwar — APPROVED** [review]
Everything looks perfect to me!

**Pull request merged**